src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
+            if (!this.store.isCompact() && stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
yeah, it's not style. The branch name might be a bit misleading but was based off the name of the issue #433, which includes a change in functionality in stating that search results should not be selectable. However favourited stocks are shown when you initially open the search, and it seems like those should be selectable. This change is to keep that existing functionality in full view that favourited stocks which are shown in search can still be selected, but nothing in compact view can be selected and in full view search results that aren't favourites already can't be selected. If we don't want the favourites in the initial search screen to be selectable, then I would say that they shouldn't be displayed when you first open search. It seems weird to open a view which shows the same favourites as in the favourites view but now you can't select them.